src/docfx/build/manifest/DependencyItem.cs

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Diagnostics;
+
+namespace Microsoft.Docs.Build
+{
+    internal class DependencyItem
```

Comment:
nit: Use tuples and stop worrying about hash code and equals

Reply:
I don't want to use tuples everywhere especially the tuple will be very complex

---
In reply to: [193676045](https://github.com/dotnet/docfx/pull/2859#discussion_r193676045) [](ancestors = 193676045)